documentation/development/release.md

```diff
@@ -9,13 +9,29 @@ This guide is for maintainers only.
 - create and push a new Git Tag for the release
   - `git tag -m release -a v4.0`
   - `git push --tags`
-- Travis-CI creates a new release on GitHub and attaches the GT binaries to it
+- run `env GITHUB_TOKEN=XXX goreleaser`
+- review the release on [GitHub](https://github.com/git-town/git-town/releases)
+  and publish it
 
 ### create a Homebrew release
 
 - fork [Homebrew](https://github.com/Homebrew/homebrew)
 - update `Library/Formula/git-town.rb`
   - get the sha256 by downloading the release (`.tar.gz`) and using
     `shasum -a 256 /path/to/file`
-  - ignore the `bottle` block. It is updated by the homebrew maintainers
+  - ignore the `bottle` block, the homebrew maintainers update it
 - create a pull request and get it merged
+
+### Arch Linux
+
+Flag the package out of date on the right hand side menu of
+[Git Town's AUR page](https://aur.archlinux.org/packages/git-town/).
+[allonsy](https://github.com/allonsy) will update the package.
+
```

Comment:
@allonsy I found a tool (goreleaser) that creates all sorts of installer for Go binaries, including `.deb` and `.rpm` packages and uploads them to the GitHub release. So I think we can finally retire your trusty tooling to create `.deb` files manually. Also we are no longer on Travis, so it wasn't running anymore anyways. Thanks for helping support Linux all the years! Git Town still needs your help maintaining the AUR package. Are you still game for that? Would be great!